test_scripts/Defects/7_0/3532_OnSCU_after_Show_widget_IGN_OFF_ON_Cycle.lua

```diff
@@ -0,0 +1,89 @@
+---------------------------------------------------------------------------------------------------
+-- Issue: https://github.com/smartdevicelink/sdl_core/issues/3532
+--
+-- Description: Check that SDL transfer `OnSystemCapabilitiesUpdated` notification from HMI to an App
+-- after IGN_OFF/IGN_ON cycle in case App sent Show RPC with templateConfiguration parameter for widget window
+-- after Show response
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) App is registered and activated (FULL level)
+-- 3) App successfully created a widget
+-- 4) Widget is activated on the HMI and has FULL level
+-- 5) IGN_OFF/IGN_ON cycle is performed
+-- 6) App is re-registered with actual HashId
+-- 7) Widget is activated on the HMI and has FULL level
+-- Steps:
+-- 1) App sends `Show` request with `templateConfiguration` for widget window
+-- SDL does:
+--  - proceed with `Show` request successfully
+--  - not send `OnSystemCapabilityUpdated` to App
+-- 2) HMI sends `OnSystemCapabilityUpdated` to SDL
+-- SDL does:
+--  - transfer `OnSystemCapabilityUpdated` notification to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WidgetSupport/common')
+
+--[[ Local Variables ]]
+local createWindowParams = {
+  windowID = 2,
+  windowName = "Name",
+  type = "WIDGET"
+}
+
+--[[ Local Functions ]]
+local function sendShow()
+  local showParams = {
+    mainField1 = "MainField1",
+    windowID = createWindowParams.windowID,
+    templateConfiguration = {
+      template = "Template1"
+    }
+  }
+  local cid = common.getMobileSession():SendRPC("Show", showParams)
+  common.getHMIConnection():ExpectRequest("UI.Show")
+  :Do(function(_, data)
+      common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+  common.getMobileSession():ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+  common.getMobileSession():ExpectNotification("OnSystemCapabilityUpdated")
+  :Times(0)
+end
+
+local function sendOnSCU()
+  local paramsToSDL = common.getOnSystemCapabilityParams()
+  paramsToSDL.appID = common.getHMIAppId()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnSystemCapabilityUpdated", paramsToSDL)
+  common.getMobileSession():ExpectNotification("OnSystemCapabilityUpdated", common.getOnSystemCapabilityParams())
+end
+
+local function createWindow(pParams, pAppId)
```

Comment:
I believe that `pAppId` parameter is redundant for this function, you can use a constant value for appId inside the function